echo-scheduler/src/main/groovy/com/netflix/spinnaker/echo/scheduler/actions/pipeline/impl/PipelineTriggerConverter.groovy

```diff
@@ -34,7 +34,8 @@ class PipelineTriggerConverter {
       triggerType          : trigger.type,
       triggerCronExpression: trigger.cronExpression,
       triggerTimeZoneId    : timeZoneId,
-      triggerEnabled       : Boolean.toString(trigger.enabled)
+      triggerEnabled       : Boolean.toString(trigger.enabled),
+      triggerRebake        : Objects.toString(trigger.rebake) // because trigger.rebake can be null
```

Comment:
it can be null ... but could we just `boolean rebake` and let it default to false?